packages/scandipwa/src/style/abstract/_media.scss

```diff
@@ -75,7 +75,13 @@
 }
 
 @mixin hoverable {
-    @media (hover: hover) and (pointer: fine) { 
+    @media (hover: hover) and (pointer: fine) {
+        @content;
+    }
+}
+
+@mixin none-pointer-device {
+    @media (pointer: coarse) or (pointer: none){
```

Comment:
Does the `or` work? Did you test it?
Because in a link I sent to you there is a different syntax.